sycl/test/union_param/union_kernel_param.cpp

```diff
@@ -8,9 +8,8 @@
 
 #include <CL/sycl.hpp>
 #include <iostream>
```

Comment:
```suggestion
#include <cstdio>
```

Reply:
iostream is not needed after change to printf.

Reply:
Yes, right. I forgot to change the header file. Thanks.